waku/v2/node/wakunode2.nim

```diff
@@ -827,6 +52,10 @@ when isMainModule:
     ./storage/message/dual_message_store,
```

Comment:
Curious if there's still a reason to keep this second group of imports are still necessary now that this module has only a single purpose?

Reply:
No, there is no technical reason behind it. I wanted this PR to keep this PR "as simple as possible". If I had moved them, it would have been harder to follow the changes.

I am planning to improve this file code, too. These imports will be merged in one of the subsequent PRs 😁